### DISCUSSION TITLE
why is the tiptap editor scrolling to top every time one line of text is removed?

### DISCUSSION THREAD

**Discussion by handhikadj** [ueberdosis/tiptap#4061] [General Questions]
it should not be scrolling like the normal textarea below it

https://github.com/ueberdosis/tiptap/assets/22522638/fbd4fa33-5d71-46ce-a46f-785260563c98

this is the reproducible: https://codesandbox.io/s/tiptap-vue-forked-gbyn0r
how do I solve this?
is this a bug on tiptap or what?

**Comment by ShravanSunder** [comment]
@handhikadj did you ever find the cause?
